streams/src/test/java/org/apache/kafka/streams/processor/internals/assignment/SubscriptionInfoTest.java

```diff
@@ -90,40 +97,39 @@ public void generatedVersion1ShouldBeDecodableByLegacyLogic() {
             1,
             1234,
             processId,
-            activeTasks,
-            standbyTasks,
-            "ignoreme"
+            "ignoreme",
+            TASK_LAGS
         );
         final ByteBuffer buffer = info.encode();
 
         final LegacySubscriptionInfoSerde decoded = LegacySubscriptionInfoSerde.decode(buffer);
         assertEquals(1, decoded.version());
         assertEquals(SubscriptionInfo.UNKNOWN, decoded.latestSupportedVersion());
         assertEquals(processId, decoded.processId());
-        assertEquals(activeTasks, decoded.prevTasks());
-        assertEquals(standbyTasks, decoded.standbyTasks());
+        assertEquals(ACTIVE_TASKS, decoded.prevTasks());
+        assertEquals(STANDBY_TASKS, decoded.standbyTasks());
         assertNull(decoded.userEndPoint());
     }
 
-
     @Test
     public void generatedVersion1ShouldDecodeLegacyFormat() {
         final LegacySubscriptionInfoSerde info = new LegacySubscriptionInfoSerde(
             1,
             LATEST_SUPPORTED_VERSION,
             processId,
-            activeTasks,
-            standbyTasks,
-            "localhost:80"
+            ACTIVE_TASKS,
+            STANDBY_TASKS,
+            "localhost:80",
+            null
```

Comment:
Q: Why do you pass in `null` here but in some other cases you pass in `TASK_LAGS`?